back/app/routes.py

```diff
@@ -159,8 +154,10 @@ def delete_all():
 
                         file_path = os.path.join(backup_dir, f"bk{i}.json")
 
-                        with open(file_path, "w") as f:
+                        try:
                                 json.dump(data_list, f)
+                        except base64.binascii.Error:
+                                return jsonify({'error': 'try again after 24 days'}), 400
                         db.data.drop()
                         return jsonify({'message': 'All complaints deleted successfully'})
         except Exception as e:
```

Comment:
Do not return the `e` it self but rather than that return `something went wrong`